guides/common/attributes-base.adoc

```diff
@@ -196,3 +196,4 @@
 :project-package-install: dnf install
 :project-package-remove: dnf remove
 :project-package-update: dnf upgrade
+:server-package-install: {project-package-install}
```

Comment:
```suggestion
// Foreman Server and Smart Proxy Server platform
:server-package-install: {project-package-install}
```

It _needs_ a comment to differentiate between line 155 and 192. However, my comment is just a suggestion.